pkg/apis/tenancy/v1alpha1/types.go

```diff
@@ -399,9 +425,19 @@ const (
 	// WorkspaceInitializedInitializerExists reason in WorkspaceInitialized condition means that there is at least
 	// one initializer still left.
 	WorkspaceInitializedInitializerExists = "InitializerExists"
-	// WorkspaceInitializedAPIBindingNotBound reason in WorkspaceInitialized condition means that at least
-	// one APIBinding is not yet bound to the workspace.
-	WorkspaceInitializedAPIBindingNotBound = "APIBindingNotBound"
+
+	// WorkspaceAPIBindingsInitialized represents the status of the initial APIBindings for the workspace.
```

Comment:
I'm not sure this makes sense to live here, as it assumes a particular initializer. Initializers are free to post their own conditions with whatever prefix & AFAIR we roll them up

Reply:
i can move it into the controller package